README.md

```diff
@@ -1 +1,56 @@
-# scm-router
\ No newline at end of file
+# SCM Router
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> A generic scm plugin that routes builds to a specified scm
+
+A scm is interface for interacting with a source control management system.
+
+i.e. Github, Bitbucket, Gitlab
+
+The scm router will allow multiple scms to be used in a Screwdriver cluster.
+
+## Usage
+
+```bash
+npm install screwdriver-scm-router
+```
+
+### Interface
+
+It will initialize any routers specified in the [default.yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L123-L156) under the `scm` and `scms` keyword. At least one of `scm` and `scms` is required.
+
+**Example scm yaml section:**
+```
+scm:
+    plugin: github
+    github:
+        oauthClientId: YOU-PROBABLY-WANT-SOMETHING-HERE
+        ......
+scms:
```

Comment:
Would there be a case where we want both of these options present? Should we just make users specify **either** `scm` **or** `scms`? 

**I actually think it might be better to just have 1 case: `scms`**. If they have 1 scm then the `scms` would be an array of 1 element. That way our logic would be much simpler. And I think as a user, it's easier to use as well.